errbot/core_plugins/chatRoom.py

```diff
@@ -25,6 +25,8 @@ def callback_connect(self):
             self.connected = True
             for room in self.bot_config.CHATROOM_PRESENCE:
                 self.log.debug('Try to join room %s' % repr(room))
+                if 'password' not in self.bot_config.BOT_IDENTITY.keys():
+                    self.bot_config.BOT_IDENTITY['password']=None
```

Comment:
My bad, this lines keeps for a previous local iteration of the code, I will remove it.